tests/test_scripts/test_solph/test_non_convex_investment_offset/test_non_convex_investment_offset.py

```diff
@@ -0,0 +1,102 @@
+"""
+This example compares a cheap and an expensive heat pump, with the latter
+adding a fixed invest cost offset to the objective function.
+
+This file is part of project oemof (github.com/oemof/oemof). It's copyrighted
+by the contributors recorded in the version control history of the file,
+available from its original location
+oemof/tests/test_scripts/test_solph/test_non_convex_investment_offset/test_non_convex_investment_offset.py
```

Comment:
This formulation is not advised. You should just give your names here.

```suggestion
SPDX-FileCopyrightText: Malte Fritz
SPDX-FileCopyrightText: Jonas Freißmann
```

Reply:
We have originaly oriented ourselves to the other (older) test files, but now we changed it according to your suggestion.